scripts/build.sh

```diff
@@ -9,4 +9,5 @@ if ! [ -x "$(command -v bikeshed)" ] || [ "$1" = "--upgrade" ]; then
     pip install bikeshed
 fi
 
+bikeshed update
```

Comment:
Is this for testing or still required to pass CI? This is a slow step so it's not great to run locally every time.

Reply:
Required to pass CI; the version of bikeshed we're installing doesn't define `navigable` at the moment.

I suggest that for local use the CI version of this script isn't very useful since you don't want to always update or die on warnings, so we should stop encouraging that and instead suggest e.g. `bikeshed serve`.